### DISCUSSION TITLE
Location of metadata.json

### DISCUSSION THREAD

**Discussion by RobCranfill** [wmerkens/vscode-circuitpython#12] [General]
I'm just trying to understand the layout of this project, and am confused at the README file's comment

>  The metadata.json file contains 550 boards

I can't find said file.

(I hope this isn't a stupid question!)

/rob

**Comment by wmerkens** [comment]
The location of the files in the boards folder on a Linux box is located here.

~/.vscode/extensions/wmerkens.vscode-circuitpython-v2-0.2.8/boards

In the GitHub code, it is not there because it's in the .gitignore file; if you build the extension, it gets created.
